common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -0,0 +1,651 @@
+                        poseStack,
+                        color.withAlpha(30),
+                        overlay.getRenderX(),
+                        overlay.getRenderY(),
+                        0,
+                        overlay.getWidth(),
+                        overlay.getHeight());
+
+                int xOffset =
+                        switch (overlay.getRenderHorizontalAlignment()) {
+                            case Left -> 3;
+                            case Center -> 0;
+                            case Right -> -3;
+                        };
+
+                int yOffset =
+                        switch (overlay.getRenderVerticalAlignment()) {
+                            case Top -> 5;
+                            case Middle -> 0;
+                            case Bottom -> -5;
+                        };
+
+                FontRenderer.getInstance()
+                        .renderTextWithAlignment(
+                                poseStack,
+                                overlay.getRenderX() + xOffset,
+                                overlay.getRenderY() + yOffset,
+                                new TextRenderTask(
+                                        overlay.getTranslatedName(),
+                                        new TextRenderSetting(
+                                                overlay.getWidth(),
+                                                color,
+                                                FontRenderer.TextAlignment.fromHorizontalAlignment(
+                                                        overlay.getRenderHorizontalAlignment()),
+                                                FontRenderer.TextShadow.OUTLINE)),
+                                overlay.getRenderedWidth() + xOffset,
+                                overlay.getRenderedHeight() + yOffset,
+                                overlay.getRenderHorizontalAlignment(),
+                                overlay.getRenderVerticalAlignment());
+            }
+
+            if (isMouseHoveringOverlay(selectedOverlay, mouseX, mouseY) && selectionMode == SelectionMode.None) {
+                RenderUtils.drawTooltipAt(
+                        poseStack,
+                        mouseX
+                                - RenderUtils.getToolTipWidth(
+                                        RenderUtils.componentToClientTooltipComponent(HELP_TOOLTIP_LINES),
+                                        FontRenderer.getInstance().getFont()),
+                        mouseY,
+                        100,
+                        HELP_TOOLTIP_LINES,
+                        FontRenderer.getInstance().getFont(),
+                        false);
+            }
+        }
+
+        super.render(poseStack, mouseX, mouseY, partialTick); // This renders widgets
+    }
+
+    @Override
+    public boolean mouseClicked(double mouseX, double mouseY, int button) {
+        if (testMode) return super.mouseClicked(mouseX, mouseY, button);
```

Comment:
Does this mean that in test mode you could be clicking around in the background UI? How well does that work? Sounds a bit scary.

Reply:
Not the background GUI, but the Screen code, which handles button clicks. If we don't have this, we can't interact with the buttons.